src/util/ArraysEqual.js

```diff
@@ -2,13 +2,5 @@ module.exports = function arraysEqual(a, b) {
   if (a === b) return true;
   if (a.length !== b.length) return false;
 
-  for (const itemInd in a) {
-    const item = a[itemInd];
-    const ind = b.indexOf(item);
-    if (ind) {
-      b.splice(ind, 1);
-    }
-  }
-
-  return b.length === 0;
+  return a.every(e => b.includes(e));
```

Comment:
If there is black magic going on, I would very much like to understand it :thinking: